### PR TITLE
Tooltip on profile page overhaul (GSI-860)

### PR DESCRIPTION
**Pull request by ac-jorellanaf**
Changed look of tooltip icon
Info icons now on the right
Added more information for tooltips
Fixed access request mocks ending access before May this year
Fixed display issue for dataset access section when dataset could not be matched with a currently valid access request

### CODE REVIEW

**Review by Cito — CHANGES_REQUESTED** [review]

**Review by Cito — CHANGES_REQUESTED** [review]
The tooltips look much better now, thanks.

File: src/components/profile/profile.tsx [hunk 1]
```diff
@@ -328,9 +351,24 @@ const Profile = () => {
...
-              &#9432;
-            </abbr>
+            <OverlayTrigger
+              overlay={
+                <Tooltip id={"ivas"}>
+                  Addresses used to verify your account's identity in order to
+                  access and request access to datasets. Adding new contact
+                  address is done via the relevant button at the end of this
+                  section. After a new address is added, verification of this
+                  address via the data steward must be manually requested.
```
[the remark this answers is not in the document]
Comment (ac-jorellanaf) [reply src/components/profile/profile.tsx]: Done.

File: src/mocks/data.js [hunk 2]
```diff
@@ -78,13 +78,13 @@ export const accessRequests = [
     full_user_name: "Dr. John Doe",
     email: "j.jdoe@home.org",
     request_text: "This is a test request for dataset TEST00002000001.",
-    access_starts: "2023-05-12T15:00:00.000Z",
-    access_ends: "2024-05-12T14:59:59.000Z",
+    access_starts: "2024-05-12T15:00:00.000Z",
```
[the remark this answers is not in the document]
Comment (ac-jorellanaf) [reply src/mocks/data.js]: Good idea! Done

File: src/mocks/data.js [hunk 3]
```diff
@@ -78,13 +78,13 @@ export const accessRequests = [
...
+    access_ends: "2025-05-12T14:59:59.000Z",
     request_created: "2023-05-11T12:04:02.000Z",
     status: "allowed",
     status_changed: "2023-05-19T12:04:03.000Z",
     changed_by: "j.doe@ghga.de",
-    iva_id: "TEST00014000001",
+    iva_id: "783d9682-d5e5-4ce7-9157-9eeb53a1e9ba",
```
[the remark this answers is not in the document]
Comment (ac-jorellanaf) [reply src/mocks/data.js:91]: Done I think

**Review by Cito — APPROVED** [review]
The wide tooltips look better now, but I think making the text left aligned will make them even more readable.

**Review by Cito — APPROVED** [review]

**Pull request merged**